tests/integration/test_interface.py

```diff
@@ -99,7 +99,7 @@ def test_generate_dataset_from_sample_and_source(
         ).mean()
         # we special-case a few sparse columns that have larger differences
         if dataset_name == DATASETS.cps.name and col == COLUMNS.unit_number.name:
-            rtol = 0.21
+            rtol = 0.30
```

Comment:
This is caused by the umpdate to miswrite numerics which is a main source of noise for unit number column.

Reply:
Does it somehow make sense that this increased after stripping?

Reply:
This is the way I was thinking of it but it still is confusing to me - we are noising the same amount for this column with all our noise types here but we now have less characters to noise but in miswrite numerics we DO NOT noise empty strings.  I would have thought similar to what @rmudambi expects.  My guess is this isn't being caused by miswrite numerics but instead is phonetic or ocr and the shorter strings make it more likely for a row to be chosen as we saw with middle initial.